common/src/main/resources/META-INF/native-image/handlers/reflect-config.json

```diff
@@ -0,0 +1,552 @@
+[
```

Comment:
Also one thing I was wonder was if we should really put this in common ? Shouldn't we have the config with the right entries in each artefact that has such a handler ?

Reply:
That's a good point - the reason why I put them under the common directory is because these would be available to every project, so we would only have to maintain it in one place. During the image build, only handlers that are reachable (both available on the classpath and used in the application) would be included

I'm looking into the reflections approach for automatically generating this metadata now - if this approach ends up working, I could split the generated metadata on a per-project basis